### ISSUE TITLE
Code fields not auto display in Tabs

### ISSUE THREAD

**Issue by bolechen** [eminiarts/nova-tabs#23]
- Nova Version: 1.3.1
- Laravel Version: 5.6.39

** Step to reproduce **
```php
    /**
     * Get the fields displayed by the resource.
     *
     * @param \Illuminate\Http\Request $request
     *
     * @return array
     */
    public function fields(Request $request)
    {
        return [

            new Tabs('Tabs', [
                'One' => [
                    Text::make('youzan_client_secret')->hideFromIndex()->rules('required'),
                ],

                'Two' => [
                    Code::make('Json', 'meta')->json(JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE)->rules('required'),
                ],
            ]),
        ];
    }
```
When I change to tab Two, the Json code field is empty, but when i click to focus, the content is back.

**Screenshots**
![image](https://user-images.githubusercontent.com/195015/53710340-add5a180-3e77-11e9-9a4b-29cd58f276ab.png)
<img width="1538" alt="wx20190304-121728 2x" src="https://user-images.githubusercontent.com/195015/53710341-b037fb80-3e77-11e9-9cd7-14b1f24e3484.png">

**Comment by bolechen** [comment]
This is the screencast in gif
![untitled](https://user-images.githubusercontent.com/195015/53710773-ccd53300-3e79-11e9-8100-ab6c279125d6.gif)

**Comment by tanthammar** [comment]
Having the same issue

**Comment by bajramemini** [comment]
Not really sure what is causing this issue. 

Anyone has an idea or has figured it our?

Regards

**Comment by novocent** [comment]
I have the same problem with the markdown field. If I click to focus or resize the browser window the content appears. If I inspect the page prior the the click or resize :
`<div class="CodeMirror-code" role="presentation"></div>` is blank. After, the content and and html markup is loaded into this div.

**Comment by RVxLab** [comment]
Due to the age of this issue and my attempt to clean everything up I'm closing this. If this is still a problem for you, please re-open this or create a new one.

**Comment by bolechen** [comment]
> Due to the age of this issue and my attempt to clean everything up I'm closing this. If this is still a problem for you, please re-open this or create a new one.

This issue is still in the newest version.

**Comment by RVxLab** [comment]
> This issue is still in the newest version.

Newest version of Nova or this package?

**Comment by bolechen** [comment]
> > This issue is still in the newest version.
> 
> Newest version of Nova or this package?

Also, in my case i test in Nova v3.12.1 & nova-tabs 1.2.3, If more information is needed, I'm happy to assist!

**Comment by RVxLab** [comment]
If you could send an up to date screenshot and/or screencast that'd be fantastic. I'll try to take a look over the week.

**Comment by bolechen** [comment]
> If you could send an up to date screenshot and/or screencast that'd be fantastic. I'll try to take a look over the week.

This is the screencast in gif, hope helpful, thx for the work.

![Screen Recording 2020-10-19 at 18 52 36](https://user-images.githubusercontent.com/195015/96444319-a4fa3500-1240-11eb-935d-43334630fe70.gif)

**Comment by RVxLab** [comment]
This is definitely helpful, thank you. I'll investigate this.

**Comment by RVxLab** [comment]
I had a look, this seems to be an issue within CodeMirror. By the looks of it, when the field is not visible after the initial load the contents are not visible.

https://stackoverflow.com/questions/8349571/codemirror-editor-is-not-loading-content-until-clicked

Ideally this should be fixed in Nova and I'll end up making an issue there after I come home from my day job. Until then, can you please try it with this branch? https://github.com/eminiarts/nova-tabs/tree/bugfix/issue-23

It's based on the current master and thus has a few bugfixes planned to be released with 1.3.0. Please let me know if this fixes it for you.


```
"eminiarts/nova-tabs": "dev-issue-23",
```

```
composer update eminiarts/nova-tabs
```

**Comment by bolechen** [comment]
I test this version local, it works prefect for me now.

Good job~

**Comment by RVxLab** [comment]
Hi @bolechen 

This fix has been released in 1.3.0. Thanks for your bug report and your screenshots.
